examples/java-matter-controller/java/src/com/matter/controller/Main.java

```diff
@@ -82,14 +85,27 @@ private static void registerCommandsPairing(
   }
 
   public static void main(String[] args) {
-    /* TODO: uncomment when SDK integration is done
+    System.setProperty("java.library.path", "../lib/jni");
+
+    Field fieldSysPath = null;
+    try {
```

Comment:
Some comments on what this does and why would help.

I could find some description of this at http://stackoverflow.com/a/24988095  ... but do we want this? At least we could comment for alternatives possible - I see many java apps have some form of launcher that set the right paths.

Reply:
From the stackoverflow thread, it looks setting the java.library.path directly within the Java program is a workaround

Remove this part and use java -Djava.library.path=../lib/jni -jar java-matter-controller to set the correct natives directory at this moment.